modules/nf-core/happy/ftxpy/tests/main.nf.test

```diff
@@ -111,8 +109,7 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
```

Comment:
Can you just put the process.out on one line (for consistency with other modules)